src/src.pro

```diff
@@ -56,6 +56,7 @@ CONFIG(release, debug|release) {
 include(../version.pri)
 
 # Qt defines
+DEFINES += QT_DISABLE_DEPRECATED_BEFORE=0x050e00
```

Comment:
Can you use `QT_VERSION` instead?

Reply:
> Is it possible to use `QT_VERSION` instead?

What do you mean?
If you mean `QT_VERSION_CHECK(5, 14)` then No, it is impossible here.

Reply:
I mean will [`QT_VERSION`](https://doc.qt.io/qt-5/qtglobal.html#QT_VERSION) expand to hex number?

Reply:
> I mean will [`QT_VERSION`](https://doc.qt.io/qt-5/qtglobal.html#QT_VERSION) expand to hex number?

But why? We do not want to disable functions that are deprecated in current Qt version used during compilation. We want to specify a specific version - Qt 5.14.

Reply:
>If you mean QT_VERSION_CHECK(5, 14) then No, it is impossible here.

OK, forget it.